docs/code-snippets/outlook-snippets.yaml

```diff
@@ -340,6 +340,30 @@ Office.AppointmentRead.dateTimeModified:
     var dateTimeModified = Office.context.mailbox.item.dateTimeModified;
     console.log("Date and time modified: " + dateTimeModified);
 Office.AppointmentRead.displayReplyAllForm:
+  - |-
+    // Reply with a body, file attachment, item attachment, and a callback.
+    Office.context.mailbox.item.displayReplyAllForm(
+    {
+        'htmlBody' : 'hi',
+        'attachments' :
+        [
+            {
+                'type' : Office.MailboxEnums.AttachmentType.File,
+                'name' : 'squirrel.png',
+                'url' : 'http://i.imgur.com/sRgTlGR.jpg'
```

Comment:
What's the policy on non-MSFT links?

Reply:
I'll dig into this and make a change subsequently if need be.